core/src/ParaGridIO.cpp

```diff
@@ -152,10 +149,9 @@ ModelState ParaGridIO::getModelState(const std::string& filePath)
             for (netCDF::NcDim& dim : varDims) {
                 dimKey += dim.getName();
             }
+            // Skip invalid dimension keys
             if (!dimensionKeys.count(dimKey)) {
-                throw std::out_of_range(
-                    std::string("No ModelArray::Type corresponds to the dimensional key ")
-                    + dimKey);
```

Comment:
Note that I needed to rework this now that groups have been removed.

Reply:
I just thought I'd point out that the way these are processed had changed to the reviewer :)